build/cli.js

```diff
@@ -180,7 +180,7 @@ async function buildDocuments(files = null) {
 
     if (!options.noProgressbar) {
       progressBar.increment();
-    } else {
+    } else if (!quiet) {
```

Comment:
Perhaps worth to think about log levels and have an environment controlling the minimum level being logged …